src/main/java/Duke.java

```diff
@@ -1,10 +1,55 @@
+import java.util.Scanner;
+import java.time.format.DateTimeParseException;
+
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    private Scanner in = new Scanner(System.in);
+    private CompilationOfFiles storage;
```

Comment:
As mentioned earlier, how about thinking of changing the name CompilationOfFiles to a shorter one?